webpack/assets/javascripts/react_app/components/common/DateTimePicker/index.js

```diff
@@ -0,0 +1,4 @@
+import DateTimePicker from './DateTimePicker';
```

Comment:
this component does not connected to redux, so `index` file is redundant, based our conventions.

Reply:
removed, thanks